product_configurator/models/product_config.py

```diff
@@ -207,8 +207,8 @@ def _compute_template_attribute_ids(self):
     )
     value_ids = fields.Many2many(
         comodel_name="product.attribute.value",
-        id1="cfg_line_id",
-        id2="attr_val_id",
+        column1="cfg_line_id",
+        column2="attr_val_id",
```

Comment:
@Chandresh-OSI Mind that this fix might break your dev/test database. To recover I had to uninstall and the reinstall the module.

Reply:
Noted.